service/automatic_run_experiment.py

```diff
@@ -54,7 +54,9 @@
     'libpcap_fuzz_both',
     'mbedtls_fuzz_dtlsclient',
     'openssl_x509',
-    'php_php-fuzz-parser',
+
+    # Temporarily disabled due to out-of-memory issues with clang coverage.
```

Comment:
Definitely shouldn't come in this PR, but I think we should switch to searching for the benchmarks instead of hardcoding a list. We only hardcoded when we had benchmarks like perl that didn't work. Instead we can have a list of benchmarks we ignore.

Reply:
Yes right, will do it next week.